misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/util/Constants.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year needs to be 2021, check in other places as well.

Reply:
Addressed with 5c09fd7